provider/kv.go

```diff
@@ -162,6 +162,9 @@ func (provider *Kv) list(keys ...string) []string {
 func (provider *Kv) listServers(backend string) []string {
 	serverNames := provider.list(backend, "/servers/")
 	return fun.Filter(func(serverName string) bool {
+		if _, err := provider.kvclient.Get(fmt.Sprint(serverName, "/url")); err != nil {
```

Comment:
> If that's feasible, I think we should at least log unexpected errors.

Yes, that is properly best. Something like `log.Debugf("Unexpected error %s", err)`? or what do you think the error sentence should be? Naming stuff and so on is hard :)

> Could we have a quick test please? :)

Of course, should be pretty straight forward. :)